src/components/Footer/Footer.jsx

```diff
@@ -6,6 +6,7 @@ import {
   FaDonate,
   FaMedium,
   FaDiscourse,
+  FaYoutube
```

Comment:
```suggestion
  FaYoutube,
```

Missing trailing comma

Reply:
I'll add the trailing comma and run the linter. Thanks for your feedback.